backend/director/agents/index.py

```diff
@@ -53,15 +146,22 @@ def run(
                 self.videodb_tool = VideoDBTool()
             else:
                 self.videodb_tool = VideoDBTool(collection_id=collection_id)
-            self.output_message.actions.append(f"Indexing {index_type} for video")
+            self.output_message.actions.append(f"Indexing {index_type} for video..")
             self.output_message.push_update()
 
             if index_type == "spoken_words":
                 self.videodb_tool.index_spoken_words(video_id)
 
             elif index_type == "scene":
-                index_id = self.videodb_tool.index_scene(video_id)
-                scene_data = {"scene_index_id": index_id}
+                scene_index_id = self.videodb_tool.index_scene(
+                    video_id=video_id,
+                    extraction_config={"threshold": 20, "frame_count": 4},
```

Comment:
This is also harcoded? can we make it dynamic too if user wants to change it by instructions.

Reply:
We have another open issue  https://github.com/video-db/Director/issues/51 to make it dynamic. Will do the changes later.